src/quo/components/utilities/token/loader.cljs

```diff
@@ -3,12 +3,13 @@
   (:require [clojure.string :as string]))
 
 (def ^:private tokens (loader/resolve-tokens))
+(def ^:private safe-lower-case (comp string/lower-case str))
 
 (defn- get-token-image*
   [token]
   (let [token-symbol (cond-> token
                        (keyword? token) name
-                       :always          (comp string/lower-case str))]
+                       :always          safe-lower-case)]
```

Comment:
I am curious, isn't that the same thing?

Reply:
I am curious too!

Reply:
The bug is actually due to passing the `token` to the `comp` macro (due to using `cond->`) instead of performing the composite function on it.

When we use `cond->`, it passes the `token` as the first argument to `comp`, which includes the token as one of the composite functions, rather than performing the result composite function on the token.

Moving it to separate variables ensures it.

Reply:
Thanks @smohamedjavid !

Reply:
That's such a fine detail.